diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/LineLength.kt

```diff
@@ -63,7 +64,7 @@ class LineLength : Rule("line-length") {
     // fixme json method
     private fun isKDocValid(node: ASTNode): Boolean {
         return try {
-            URL(node.text).toURI()
+            URL(node.text.split("\\s".toRegex()).last { it != "" }).toURI()
```

Comment:
I think you should trim here instead: if you trim `node.text` first, then last element of split should not be empty. Also, I think it is better to use `it.isNotEmpty()` instead of comparison.

Reply:
I should trim in `checkLength` to take `KDOC_TEXT` not `WHITE_SPACE`

Reply:
Ok, got it. Also, shouldn't you be using `\\s+` as a regex here to split on multiple spaces too?